.github/workflows/run_test_suite.yaml

```diff
@@ -67,11 +67,24 @@ jobs:
         with:
           override: true
           profile: minimal
+          components: rustfmt, clippy
           toolchain: stable
       - name: 'Install environment packages'
         run: |
           sudo apt-get update -qqy
           sudo apt-get install jq protobuf-compiler cmake
+      - name: 'Check Format'
```

Comment:
Let's run this as a separate job. That way it will get its own status line, so we can see if both A) lints pass and B) tests pass.

The jobs can run in parallel anyway.